Cmdline option to open a file in write mode (#2216)
radareorg/cutter: docs/source/user-docs/command-line.rst

.. option:: -i <file>

   Run script file

> .. option:: -w, --writemode <file>

Reviewer: ```suggestion
.. option:: -w, --writemode
```

>    Open <filename> in write mode, not in the default read-only mode.
>    When used together with -A/--analysis <level>, will open a file directly
>    in write mode without any further dialog or confirmation.

Reviewer: ```suggestion
   Open <file> in write mode, instead of the default read-only mode.
   When used together with -A/--analysis <level>, it will open a file directly
   in write mode without any further dialog or confirmation.
```

Reviewer 2: `-w`, `--writemode` are actually flags and don't accept `<file>` argument so it shouldn't be listed here.

Reviewer 2: And example for Cutter at the top uses `<filename>`

Reviewer: @tobigrimm if the suggestions look good to you, you can click "commit sugestion" and we will merge :)
